README.md

```diff
@@ -366,6 +384,14 @@ Once you've done this, please feel free to open a pull request adding your scrip
 
 Thanks for contributing back! Our hope is that eventually this repo will contain many useful generic scripts for common maintenance and upgrading tasks that everyone can leverage to save time.
 
+## Building the binary from source
+
+Clone this repository and then run the following command from the root of the repository: 
+
+`go build -o git-xargs ./cmd/git-xargs`
```

Comment:
`go build` should be enough here. `cmd` doesn't exist anyway. Also, use code fences (three backticks) for code blocks.

Reply:
:+1:  0689413